src/benchmarks/real-world/PowerShell.Benchmarks/Engine.Compiler.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#if NET6_0
```

Comment:
Do we want to benchmark these benchmarks on newer dotnet versions than Net6.0?

Reply:
Good catch, yes we probably want to run these on versions newer than net6.0.

Reply:
Just to make sure that I understand. In our lab, we always run against current which is `.net8`. Leaving it as .NET 6 will not execute this benchmark?

Reply:
That is correct as a whole. Benchmarks in the main branch are only run against whatever latest is. Each release we make a branch to correspond to that release and hold the tests to be run against that release while it is in support, but we don't generally backport new tests to these branches.

Reply:
It shouldn't. If the project is compiled to a target other than NET6, then these benchmarks will not be included.